content/pages/library.md

```diff
@@ -0,0 +1,35 @@
+Title: SADev Library
+Date: 7-19-2019
+Slug: library
+Authors: Jason Gutierrez
+Summary: A curated list of recommended books for the SADev community
+
+# Library
+
+This library is a collection of links to books and articles that have been recommended by other members of the community. The categories will grow as
+more recommendations are added. If there is a book which is free and open to the public the pdf version will be provided for our community members.
+
+## Categories
+
+**[AI & Machine Learning](#ai-&-machine-learning)**  
+**[Coding Architecture](#coding-architecure)**  
+**[Philosophical](#Philosophical)**  
+**[UI/UX](#uxui)**  
+
+## AI & Machine Learning
+
+## Coding Architecture
+
+**[ATDD by Example: A Practical Guide to Acceptance Test-Driven Development by Markus Gartner](https://www.goodreads.com/en/book/show/13705173-atdd-by-example)**  
```

Comment:
Since these are being recommended by community members (per line 9), then maybe it would make sense to include who recommended it here.  That way you'd know who to talk to if you want to discuss the book or maybe borrow a physical copy.